tests/test_errors.py

```diff
@@ -696,6 +708,18 @@ def finalize(self, cfg: Any) -> None:
         ),
         id="typed_DictConfig:assign_with_invalid_value,float",
     ),
+    param(
+        Expected(
+            create=lambda: DictConfig({"bar": BytesNode(b"abc123")}),
+            op=lambda cfg: cfg.__setattr__("bar", "x"),
+            exception_type=ValidationError,
+            msg="Value 'x' of type 'str' could not be converted to Bytes",
```

Comment:
```suggestion
            op=lambda cfg: cfg.__setattr__("bar", 123.4),
            exception_type=ValidationError,
            msg="Value '123.4' of type 'float' could not be converted to Bytes",
```
This should fix one of the failing tests.
Calling `__setattr__("bar", "x")` no longer raises a `ValidationError` because we are using the `str.encode` method now (thanks to commit https://github.com/omry/omegaconf/pull/845/commits/ec80f6ff433f0ec547cf9231e52fc3f11ddecd6f).

Reply:
Resolved in https://github.com/omry/omegaconf/pull/845/commits/ee470da79e28057fe5b7ae72060088c30bd2babf